src/saving/SaveList.tscn

```diff
@@ -104,7 +112,6 @@ WindowTitle = "SELECTED_SAVE_IS_UPGRADEABLE"
 DialogText = "SAVE_IS_UPGRADEABLE_DESCRIPTION"
 
 [node name="SaveUpgradeFailedDialog" parent="MarginContainer/VBoxContainer" instance=ExtResource( 4 )]
-visible = false
```

Comment:
This may be unintended change.

Reply:
Godot seems to remove any lines containing `visible = false`.
Cause might be Godot interpreting missing `visible`-values as false.
Re-Adding that in line in the SaveList.tscn file, results in Godot removing it.

Reply:
I've seen this type of change many times before. This is caused by setting a node that we have purposefully hidden, to be visible. The default value for a node is to be visible, meaning that when Godot removes this line, that's the same as making the node visible. So you need to in the Godot editor find this node and click it back to hidden state, or use git to revert this line.

Reply:
Node was already hidden, got removed anyway.
Used git to revert the line.